components/Form.tsx

```diff
@@ -1,11 +1,55 @@
+import Image from "next/image";
+import DogImage from "../public/Form images/dog_form.png";
+import CatImage from "../public/Form images/cat_form.png";
+
 const Form = () => {
-    return <form className={`flex flex-col justify-between items-center h-full w-full text-white mt-10`}>
-        Contact us to book a slot
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <button type="submit" className={`text-orange rounded bg-black pt-2 pb-2 pl-2 pr-2`}>Paw us</button>
-    </form>
-}
-export default Form;
\ No newline at end of file
+  return (
+    <div className="flex flex-col h-full w-full mt-10 px-6 md:px-24 text-black">
+      <div className="bg-orange rounded-3xl p-5 flex flex-col shadow-xl">
+        <h2 className="lg:text-3xl md:text-2xl font-extrabold mb-4">Контактирајте нѐ!</h2>
+        <p className="mb-6">Поставете прашање или она што Ве интересира</p>
```

Comment:
Just noticed this, the font should be bolder.
![image](https://github.com/learnhubmk/skopje-paws/assets/11035261/1c92e5ab-1fbf-4bb3-ba99-ffa86c58b910)